taxonomies/ingredients.txt

```diff
@@ -13499,6 +13499,19 @@ de:Salz aus der Camargue
 fr:Sel de camargue
 nl:Zout uit de Camargue
 
+<en:sea salt
+en:sel gris, grey salt
+fr:sel gris, sel marin gris, sel gris marin
+wikipedia:en:https://en.wikipedia.org/wiki/Sel_gris
+wikidata:en:Q5493196
+# ingredient/fr:sel-gris-marin has 32 products in french @2020-04-03
+
+<en:sel gris
+<en:Salt from Guérande
+fr:sel gris de Guérande
```

Comment:
Let's add the English name. 
We have 3 products https://world.openfoodfacts.org/ingredient/grey-salt-from-guerande

Reply:
Added.